yoga/Yoga.c

```diff
@@ -1480,14 +1487,10 @@ static void YGNodeComputeFlexBasisForChild(const YGNodeRef node,
       }
     }
 
-    YGConstrainMaxSizeForMode(YGValueResolve(&child->style.maxDimensions[YGDimensionWidth],
-                                             parentWidth),
-                              &childWidthMeasureMode,
-                              &childWidth);
-    YGConstrainMaxSizeForMode(YGValueResolve(&child->style.maxDimensions[YGDimensionHeight],
-                                             parentHeight),
-                              &childHeightMeasureMode,
-                              &childHeight);
+    YGConstrainMaxSizeForMode(
+        child, YGDimensionWidth, parentWidth, parentWidth, &childWidthMeasureMode, &childWidth);
```

Comment:
Should `YGDimensionWidth` and `YGDimensionHeight` be changed to be `YGFlexDirection*` enums instead of `YGDimension*` enums?